templates/category.twig

```diff
@@ -0,0 +1,43 @@
+{% extends "base.twig" %}
+
+{% block content %}
+
+	<div class="clearfix"></div>
+	<div class="skewed-overlay"></div>
+
+	<header class="page-header">
+		<div class="container">
+			<h1 class="page-header-title">{{ page_type.name }}</h1>
```

Comment:
this should be changed to category.name since we changed the variable that is being passed to the template
`category.name`
https://github.com/greenpeace/planet4-master-theme/pull/647/files#diff-538c51b64adf70254db21dbfa4b24e44R16